deltachat-ios/Controller/AllMediaViewController.swift

```diff
@@ -64,26 +64,14 @@ class AllMediaViewController: UIPageViewController {
         navigationItem.rightBarButtonItem = UserDefaults.standard.bool(forKey: "location_streaming") ? mapButton : nil
         navigationController?.navigationBar.scrollEdgeAppearance = navigationController?.navigationBar.standardAppearance
 
-        // select the first page that actually contains some media
-        DispatchQueue.global(qos: .userInitiated).async { [weak self] in
+        DispatchQueue.main.async { [weak self] in
```

Comment:
viewDidLoad is already only called on main thread so the dispatch seems unnecessary here?

Reply:
can still make a difference wrt responsiveness, or? but probably not needed, i agree

Reply:
Will probably make it worse because it is not deterministic when it will be executed, maybe during an animation which might show as a hiccup